pymaker/oracles.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2019 grandizzy
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from web3 import Web3
+
+from pymaker import Contract, Address, Transact
+
+
+# TODO: Complete implementation and unit test
+class OSM(Contract):
+    """A client for the `OSM` contract.
+
+    You can find the source code of the `OSM` contract here:
+    <https://github.com/makerdao/osm>.
```

Comment:
usually we provide a link to the contract's solidity: https://github.com/makerdao/osm/blob/master/src/osm.sol